graph-db/README.md

```diff
@@ -49,13 +49,54 @@ Clone the graphexp repository at https://github.com/bricaud/graphexp
 ## Serverless Applications
 There are two serverless applications that interact with the graph database:
 
-* Bootstrap
+## Bootstrap
 
   Bootstrap is a serverless application that load all collections from a CMR environment (SIT, UAT, PROD) into the graph database.
 
-* Indexer
+### Build
+```
+npm install
+```
+
+### Run
+To invoke the bootstrap function and load data into your Gremlin server from the CMR run the following command:
+```
+npm run bootstrap
```

Comment:
`bootstrap` no longer exists. You changed it to `bootstrap-local`. We need to update the description above to clarify what is being run.